asmcomp/cmmgen.ml

```diff
@@ -3184,12 +3189,12 @@ let send_function arity =
     let cached_pos = Cvar cached in
     let tag_pos = Cop(Cadda, [Cop (Cadda, [cached_pos; Cvar meths], dbg);
                               Cconst_int(3*size_addr-1)], dbg) in
-    let tag' = Cop(Cload (Word_int, Mutable), [tag_pos], dbg) in
+    let tag' = Cop(Cload {memory_chunk=Word_int; mutability=Mutable; is_atomic=false}, [tag_pos], dbg) in
     Clet (
     meths, get_mut_field obj (Cconst_int 0) dbg,
     Clet (
     cached,
-      Cop(Cand, [Cop(Cload (Word_int, Mutable), [cache], dbg); mask], dbg),
+      Cop(Cand, [Cop(Cload {memory_chunk=Word_int; mutability=Mutable; is_atomic=false}, [cache], dbg); mask], dbg),
```

Comment:
Looks like most loads are mutable and non-atomic. It might be useful to write a helper function at the top of this file to avoid verbosity:

```ocaml
let mut_load memory_chunk = Cload {memory_chunk; mutability=Mutable; is_atomic=false}
```

will help with clarity.